actors/market/tests/random_cron_epoch_during_publish.rs

```diff
@@ -16,89 +15,13 @@ const START_EPOCH: ChainEpoch = 50;
 const END_EPOCH: ChainEpoch = START_EPOCH + 200 * EPOCHS_IN_DAY;
 const SECTOR_EXPIRY: ChainEpoch = END_EPOCH + 1;
 
-#[test]
-fn cron_processing_happens_at_processing_epoch_not_start_epoch() {
-    let rt = setup();
-
-    let deal_id = generate_and_publish_deal(
-        &rt,
-        CLIENT_ADDR,
-        &MinerAddresses::default(),
-        START_EPOCH,
-        END_EPOCH,
-    );
-    let deal_proposal = get_deal_proposal(&rt, deal_id);
-
-    // activate the deal
-    rt.set_epoch(START_EPOCH - 1);
-    activate_deals(&rt, SECTOR_EXPIRY, PROVIDER_ADDR, deal_proposal.start_epoch - 1, &[deal_id]);
-
-    // cron tick at deal start epoch does not do anything
-    rt.set_epoch(START_EPOCH);
-    cron_tick_no_change(&rt, CLIENT_ADDR, PROVIDER_ADDR);
-
-    // first cron tick at process epoch will make payment and schedule the deal for next epoch
-    let deal_epoch = process_epoch(START_EPOCH, deal_id);
-    rt.set_epoch(deal_epoch);
-    let (pay, _) =
-        cron_tick_and_assert_balances(&rt, CLIENT_ADDR, PROVIDER_ADDR, deal_epoch, deal_id);
-    let duration = deal_epoch - START_EPOCH;
-    assert_eq!(duration * &deal_proposal.storage_price_per_epoch, pay);
-
-    // payment at next epoch
-    let new_epoch = deal_epoch + Policy::default().deal_updates_interval;
-    rt.set_epoch(new_epoch);
-    let (pay, _) =
-        cron_tick_and_assert_balances(&rt, CLIENT_ADDR, PROVIDER_ADDR, new_epoch, deal_id);
-    let duration = new_epoch - deal_epoch;
-    assert_eq!(duration * &deal_proposal.storage_price_per_epoch, pay);
-
-    check_state(&rt);
-}
```

Comment:
This one still seems relevant, except change it to assert that the deal is _not_ rescheduled after the first time.